@xen-orchestra/web/src/pages/vm/[id]/console.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <p v-if="!isVmConsoleAvailable" class="typo h5-medium">{{ $t('power-on-vm-for-console') }}</p>
   <VtsLayoutConsole v-else>
-    <VtsRemoteConsole :url />
+    <VtsRemoteConsole ref="consoleElement" :url />
     <template #actions>
-      <VtsActionsConsole />
+      <VtsActionsConsole :send-ctrl-alt-del="sendCtrlAltDel" />
```

Comment:
Remove `sendCtrlAltDel === undefined ? $t('coming-soon') : undefined` from the `VtsActionsConsole`. Same for the `:disable` property